client/src/components/SocialMediaForm.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+
+const SocialMediaForm = () => {
+  const [name, setName] = useState("");
+  const [url, setUrl] = useState("");
+  const [description, setDescription] = useState("");
+  const history = useHistory();
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    const socialMedia = { name, url, description };
+    const response = await fetch("/api/v1/websites", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(socialMedia),
+    });
+    if (response.ok) {
+      setName("");
+      setUrl("");
+      setDescription("");
+      history.push("/");
```

Comment:
the Redirect article in week 4 actually suggests that you DONT use history.push(), which is an older redirect pattern. Please use the Redirect component instead